<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeRequest</class>
 <widget class="QWidget" name="MakeRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Заполнение заявки на заселение</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="surname_label">
     <property name="text">
      <string>Введите фамилию</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="surname_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="name_label">
     <property name="text">
      <string>Введите имя</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="patronymic_label">
     <property name="text">
      <string>Введите отчество (при отсутствии можно опустить)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="patronymic_edit"/>
   </item>
   <item>
    <widget class="QGroupBox" name="status_groupbox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Вы студент или сотрудник?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="student_radio">
        <property name="text">
         <string>Студент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sotrudnik_radio">
        <property name="text">
         <string>Сотрудник</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="group_work_label">
     <property name="text">
      <string>Если вы студент, укажите вашу группу. Если вы сотрудник, укажите вашу должность.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="group_work_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="phone_label">
     <property name="text">
      <string>Укажите ваш номер телефона</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phone_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="mail_label">
     <property name="text">
      <string>Укажите ваш адрес элемтронной почты</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mail_edit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>Нажмите, чтобы отправить заявку на рассмотрение.</string>
       </property>
       <property name="text">
        <string>Отправить заявку</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
